go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/hyperledger/fabric-admin-sdk
 
-go 1.21
+go 1.23
 
 require (
 	github.com/IBM/sarama v1.43.1
 	github.com/Knetic/govaluate v3.0.0+incompatible
+	github.com/golang/mock v1.1.1
```

Comment:
We should be using `go.uber.org/mock` instead of the (deprecated) `github.com/golang/mock`

Reply:
[graph.txt](https://github.com/user-attachments/files/17074673/graph.txt)
I don't know why but... it seems that we have `github.com/golang/mock` as a part of go mod, after `go mod tidy` so I am going to use
```
go mod edit -replace=github.com/golang/mock@v1.1.1=github.com/uber-go/mock@v0.4.0
```
to apply `go.uber.org/mock`

Reply:
I don't see that behaviour. I wonder if you have locally some generated mocks that were created using github.com/golang/mock? If you get rid of the generated mocks, either with `find . -name '*_mock_test.go' -delete -print` or `git clean -xdf`, then run `make generate` again, you should get mocks only using go.uber.org/mock.

From this clean state, running either `go get ./...` or `go mod tidy` should produce no changes to the go.mod and go.sum files.